runtime/syntax/vimnormal.vim

```diff
@@ -0,0 +1,16 @@
+syn match normalOp /[dcrypoaxv!"#%&.-\/:<>=?@ABCDGHIJKLMNOPQRSUVWXYZgmqstz~iu]/ nextgroup=normalMod
```

Comment:
Is there any way to avoid the extra syntax file here? Maybe define it inline in the tutorial autoload? Since this is a one-time-use syntax...